### PR TITLE
Fix accessibility violations for the date picker

### PR DESCRIPTION
**Pull request by stevekinney**
- Addresses contrast issues in dark mode.
- Swaps out label-less buttons for empty elements for days outside of the month.
- Adds labels to previous and next month buttons.
- Adds previous and next text to `i18n`.

### CODE REVIEW

File: src/lib/i18n/locales/en/date-picker.ts
Comment (rossedfort) [inline src/lib/i18n/locales/en/date-picker.ts:1]: These could probably just go in the `common.ts` strings file, but this is fine too

File: src/lib/holocene/date-picker.svelte [hunk 1]
```diff
@@ -93,18 +97,21 @@
     >
       <div class="mx-3 my-2 flex items-center justify-around">
         <div class="flex items-center justify-center">
-          <button type="button" on:click={prev}
-            ><Icon name="chevron-left" /></button
+          <button type="button" on:click={prev} title={previousMonth}>
+            <span class="sr-only">{previousMonth}</span>
```
Comment (rossedfort) [inline src/lib/holocene/date-picker.svelte:101]: Is this preferred over just specifying `aria-label` on the button? I imagine it accomplishing the same thing.

File: src/lib/holocene/calendar.svelte [hunk 2]
```diff
@@ -38,20 +38,24 @@
...
-        class:disabled={!allowed}
-        class:selected={new Date(
-          date.getFullYear(),
-          date.getMonth(),
-          date.getDate(),
-        ).getTime() === new Date(year, month, value).getTime()}
-      >
-        {value || ''}
-      </button>
+      {#if value}
```
Comment (rossedfort) [inline src/lib/holocene/calendar.svelte:41]: Why are we only showing the button if it has a value? So it doesn't have an empty label? If so - maybe we should just change the empty state label down on line 54.

Comment (stevekinney) [reply src/lib/holocene/calendar.svelte:41]: Well, it's not _really_ a button at that point either. So, we really just need something to take up space.

File: src/lib/i18n/locales/en/date-picker.ts
Comment (stevekinney) [reply src/lib/i18n/locales/en/date-picker.ts:1]: I wasn't sure if it might grow with months and whatnot.

**Pull request merged**